### ISSUE TITLE
High contrast chart is barely viewable 

### ISSUE THREAD

**Issue by Norric1Admin** [ONSdigital/sdg-indicators#3099]
When the ‘Chart’ is viewed in high-contrast mode the related text has insufficient colour contrast of 4.1:1 (foreground colour: #6E6E6E, background colour: #000000). The expected minimum colour contrast ratio for text of this size is 4.5:1. When viewed in the default contrast mode however, the text related to the charts have sufficient colour contrast. This means that some users may be required to alternate between default and high-contrast modes to distinguish all of the page content which can be frustrating.

We recommend that high-contrast style sheets meet the minimum colour contrast requirements to enable users to distinguish page content without alternating between both default and high-contrast modes. Switch the high-contrast chart with the standard one.

**Comment by Norric1Admin** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Accessibility tickets](https://trello.com/c/E40VA79G/549-accessibility-tickets)

**Comment by LucyGwilliamAdmin** [comment]
## Description

## Screenshots
**Before**

**After**


## Successfully tested on...
- [] Windows: IE11
- [] Windows: Chrome
- [ ] Windows: Firefox
- [ ] Windows: Edge
- [ ] macOS: Chrome
- [ ] macOS: Firefox
- [ ] iOS: Safari 9.3
- [ ] iOS: Chrome
- [ ] Samsung: Chrome
- [ ] Samsung: Samsung Internet

## User feedback and problems encountered...

**Comment by AnnCorp** [comment]
Resource focussed on achieving AA, so won't be actioning AAA fixes currently.
